### PR TITLE
Blog post: "Our Vision for wasm-bindgen"

### PR DESCRIPTION
**Pull request by fitzgen**

### CODE REVIEW

**Comment by fitzgen** [comment]
[Rendered](https://github.com/fitzgen/rustwasm.github.io/blob/our-vision-for-wasm-bindgen/_posts/2018-06-29-vision-for-wasm-bindgen.md)

File: _posts/2018-06-29-vision-for-wasm-bindgen.md [hunk 1]
```diff
@@ -0,0 +1,108 @@
...
+[mpsc]: https://github.com/rustwasm/team/issues/163
+
+### Sharing ECMAScript Global APIs
+
+The global APIs that are available in every JavaScript environment, as defined
+by the ECMAScript standard, are being made available through the
+`wasm_bindgen::js` module. For example, the `Object.freeze` function is
+available as `wasm_bindgen::js::Object::freeze`.
+
+*But we aren't done yet, and we need more help!* Adding more of these global API
```
Comment (sendilkumarn) [inline _posts/2018-06-29-vision-for-wasm-bindgen.md]: can we link that issue over here?

File: _posts/2018-06-29-vision-for-wasm-bindgen.md [hunk 2]
```diff
@@ -0,0 +1,108 @@
...
+
+[![wasm-bindgen's new architecture with a WebIDL frontend](/images/new-wasm-bindgen-architecture.png)](/images/new-wasm-bindgen-architecture.png)
+
+Does this sound cool to you? **Check out [issues labeled "frontend:webidl" in
+the `wasm-bindgen` repository][webidl-issues] to help us build the WebIDL
+frontend!** The WebIDL frontend's code lives in `wasm-bindgen/crates/webidl`,
+and if you have any questions, don't hesitate to ping `fitzgen` in `#rust-wasm`
+on irc.mozilla.org or drop a comment in the relevant issue.
+
+Special shout out to [`@ohanar`][ohanar], who has been doing a wonderful job
```
Comment (sendilkumarn) [inline _posts/2018-06-29-vision-for-wasm-bindgen.md]: I love this one 👏

**Comment by alexcrichton** [comment]
Looks great to me!

**Comment by fitzgen** [comment]
Will aim for posting this on Monday, so if anyone else has some feedback please leave it before then :)

**Pull request merged**